df.d_init.xml

```diff
@@ -61,84 +62,96 @@
-        <uint8_t name='chasm_tile'/>
-        <uint8_t name='pillar_tile'/>
-
-        N S E W NS NE NW SE SW EW NSE NSW NEW SEW NSEW
-        <static-array name='track_tiles' type-name='uint8_t' count='15' since='v0.34.08'/>
-        <static-array name='track_tile_invert' type-name='uint8_t' count='15' since='v0.34.08'/>
-        <static-array name='track_ramp_tiles' type-name='uint8_t' count='15' since='v0.34.08'/>
-        <static-array name='track_ramp_invert' type-name='uint8_t' count='15' since='v0.34.08'/>
+    <enum-type type-name='d_init_adventurer_zview' base-type='int32_t'> bay12: AdventureZViewType
+        <enum-item name='OFF'/>
+        <enum-item name='UNHIDDEN'/>
+        <enum-item name='CREATURE'/>
+        <enum-item name='ON'/>
+    </enum-type>
 
-        <static-array name='tree_tiles' type-name='uint8_t' count='104' since='v0.40.01'/>
+    <struct-type type-name='d_init'>
+        <compound name='display'> bay12: d_init_displayst
+            <df-flagarray name='flags' index-enum='d_init_flags1'/>
 
-        <static-array name='chasm_color' type-name='int16_t' count='3'/>
+            <static-array name='nickname' count='10' index-enum='game_type'><enum type-name='d_init_nickname'/></static-array>
 
-        <compound name='wound_color'>
-            <static-array name='none' type-name='int16_t' count='3'/>
-            <static-array name='minor' type-name='int16_t' count='3'/>
-            <static-array name='inhibited' type-name='int16_t' count='3'/>
-            <static-array name='function_loss' type-name='int16_t' count='3'/>
-            <static-array name='broken' type-name='int16_t' count='3'/>
-            <static-array name='missing' type-name='int16_t' count='3'/>
-        </compound>
+            <uint8_t name='sky_tile'/>
+            <static-array name='sky_color' type-name='int16_t' count='3'/>
+            <uint8_t name='chasm_tile'/>
+            <uint8_t name='pillar_tile'/>
 
-        <int32_t/> probably IDLERS
+            N S E W NS NE NW SE SW EW NSE NSW NEW SEW NSEW
+            <static-array name='track_tiles' type-name='uint8_t' count='15' since='v0.34.08'/>
+            <static-array name='track_tile_invert' type-name='uint8_t' count='15' since='v0.34.08'/>
+            <static-array name='track_ramp_tiles' type-name='uint8_t' count='15' since='v0.34.08'/>
+            <static-array name='track_ramp_invert' type-name='uint8_t' count='15' since='v0.34.08'/>
 
-        <enum name="show_embark_tunnel" base-type='int32_t' type-name='d_init_tunnel'/>
-        <int32_t name='number_of_lower_elevations_shown' since='v0.50.01'/>
+            <static-array name='tree_tiles' type-name='uint8_t' count='104' since='v0.40.01'/>
 
-        <df-flagarray name='flags3' index-enum='d_init_flags3'/>
-        <int32_t/>
-        <int32_t/>
-        <int16_t/>
+            <static-array name='chasm_color' type-name='int16_t' count='3'/>
 
-        <padding size='6'/>
```

Comment:
Added compounds in order to get rid of this padding (which might have been incorrect on Linux), will require updating all `d_init` references.

Reply:
I see the windows size changed as well

Reply:
Yes, because the two integers "unk_5010_730" and "unk_5010_734" at the end weren't actually there (they got moved into the announcements list).